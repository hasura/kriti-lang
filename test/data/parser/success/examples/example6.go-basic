{{ if $.foo }} 1 {{ else }} null {{end }}