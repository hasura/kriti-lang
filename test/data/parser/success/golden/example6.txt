Iff
    ( SourcePosition
        { _sourceName = ""
        , _line = 1
        , _column = 1
        }
    , Just
        ( SourcePosition
            { _sourceName = "sourceName"
            , _line = 1
            , _column = 41
            }
        )
    )
    ( Path
        [
            (
                ( SourcePosition
                    { _sourceName = "sourceName"
                    , _line = 1
                    , _column = 7
                    }
                , Just
                    ( SourcePosition
                        { _sourceName = "sourceName"
                        , _line = 1
                        , _column = 8
                        }
                    )
                )
            , Obj "$"
            )
        ,
            (
                ( SourcePosition
                    { _sourceName = "sourceName"
                    , _line = 1
                    , _column = 8
                    }
                , Just
                    ( SourcePosition
                        { _sourceName = "sourceName"
                        , _line = 1
                        , _column = 11
                        }
                    )
                )
            , Obj "foo"
            )
        ]
    )
    ( Number 1.0 ) Null